scripts/test-rules.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+echoerr() {
+    RED='\033[0;31m'
+    NC='\033[0m' # No Color
+    echo -e "${RED}$@${NC}" 1>&2;
+}
+
+rule_path=$(dirname $1)
+rule_name="$(basename $1 | cut -d . -f1)"
```

Comment:
```suggestion
rule_name="$(basename ${1%.*}"
```

To avoid having issues if someone writes a rule with a dot on the filename.